core/tx_pool.go

```diff
@@ -83,6 +83,10 @@ var (
 	// than some meaningful limit a user might use. This is not a consensus error
 	// making the transaction invalid, rather a DOS protection.
 	ErrOversizedData = errors.New("oversized data")
+
+	// ErrTipAboveFeeCap is a sanity error to ensure no one is able to specify a
+	// transaction with a tip higher than the total fee cap.
+	ErrTipAboveFeeCap = errors.New("tip higher than fee cap")
```

Comment:
Please use `tip cap`, not `tip`. There's no notion of user configurable tip.

Reply:
Is this now the "official term"? Or should it be? It was called "tip" originally, even though it's a max (at least by some people). If we change it then maybe we should converge to some canonized terms.